_format/1.0/index.md

```diff
@@ -926,13 +926,13 @@ If an endpoint supports the `include` parameter and a client supplies it,
 the server **MUST NOT** include unrequested [resource objects] in the `included`
 section of the [compound document].
 
+If a server is unable to identify a relationship path or does not support
+inclusion of resources from a path, it **MUST** respond with 400 Bad Request.
+
```

Comment:
Sorry, I've got a ton of things on my plate right now.  Feel free to close the PR and resolve however you think is best.  The order just confused me and my pair for a few minutes when we were trying to go over the spec ;)

(thanks for JSON API, BTW, I'm loving it!)